spec/lib/github_linkback_spec.rb

```diff
@@ -2,6 +2,7 @@
 
 describe GithubLinkback do
   let(:github_commit_link) { "https://github.com/discourse/discourse/commit/76981605fa10975e2e7af457e2f6a31909e0c811" }
+  let(:github_commit_link_with_anchor) { "https://github.com/discourse/discourse/commit/76981605fa10975e2e7af457e2f6a31909e0c811#anchor" }
```

Comment:
We should use `github_commit_link` to ensure it's almost the same link

```suggestion
  let(:github_commit_link_with_anchor) { "#{github_commit_link}#anchor" }
```